internal/database/sqlcommon/contractevents_sql.go

```diff
@@ -35,7 +35,7 @@ var (
 		"name",
 		"outputs",
 		"info",
-		"created",
+		"timestamp",
```

Comment:
Is there a corresponding set of db migrations?

Reply:
Yes, thanks for catching this. Must have missed that one on the `git add`

Reply:
Pushed the SQLite migration. All the PostgreSQL migrations are coming a subsequent PR

Reply:
OK - just to note, there is a postgres migration for this table already. But I'm fine if the edits to that file come along with the rest of the yet-to-be-added postgres migrations.